### ISSUE TITLE
Prisha Individual Review Ticket (FRQ Showing 2)

### ISSUE THREAD

**Issue by PrishaMaiti** [rkwreck/team_flappy_bird#34]
Repl: https://replit.com/@PrishaMaiti/PrishasTests#Main.java 
Link to github source: https://github.com/PrishaMaiti/PrishasTests 
My video: https://drive.google.com/file/d/1tCDH6cSXz56yF_uUt_9Qy9D-oartjs_X/view?usp=sharing 
GitHub Wiki: https://github.com/rkwreck/team_flappy_bird/wiki/Prisha-FRQ-Showing-2-(Week-8) 
These are the things I'm required to have. You can check them off as you find them:

1. Individual Work: score individual on completion percentage.

- [x] 9 FRQs are under management (90%) - Units 2-10 code are in appropriate folders on Repl
- [ ] At least 2 FRQs with input and output (10%) - Book FRQ and Animal FRQ of Unit 9
- [x] Tangible evidence - my video and link to repl and github source and github wiki

**Comment by chenxin-chex** [comment]
(-10%) No FRQs have input
(-10%) There is no code for FRQ3
(+80%) The rest of the FRQs are in replit, and some are running. Github wiki clearly shows learning for all the FRQs, and the video does a good job explaining the code. The replit has a github source that is linked.

Score: 8/10

**Comment by PrishaMaiti** [comment]
Comparison Analysis and Reflection from Cross over with Chenxin:

1. FRQ 9 (Animal). She created all aspects of code execution within a main method of one of her FRQ 9 files, while I did everything on Main.java and left the FRQ classes themselves as generic. This was both a plus improvement for me, as it kept my files on Replit more organized, while it was also a minus improvement for me because it made my Main.java file very cluttered. Next time, I should find a way to organize my Main.java in a similar style that Chenxin did with separate files.
2. FRQ 8. Chenxin's code used keyboard input for the output while I just used output if the code started running. I did have code for FRQ 8, but it didn't run correctly, so there wasn't really any output in the first place, as it wasn't one of the FRQs I chose to show so I commented it out (minus improvement for me).
3. FRQ 9 (Book). Chenxin used a return type for printBookInfo even though it was supposed to be void, while I just printed what Chenxin decided to return (plus improvement for me).

Reflection:
My FRQs for the most part do not seem to be running correctly, however the ones that do demonstrate that I understand the concepts. None of my FRQs have a user input for output, but all of my FRQs do run as soon as the user uses keyboard input to select which FRQ they want to see. My wiki page shows that I understand the basic ideas of each unit, but I still struggle with being able to correctly put them to use. If there's anything I learned from Chenxin, is that I should make my FRQs more interactive so that they have inputs and outputs.
